### PR TITLE
Pageswithnavigation

### PR DESCRIPTION
**Pull request by ttaannjjaa**

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -13,10 +15,10 @@ export const parameters = {
 
 export const decorators = [
 (Story) => (
-  <>
+  <MemoryRouter>
```
Comment (Quaquaro) [inline .storybook/preview.js:18]: I would have forgotten that in my case.

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
+import LikedFoodPage from './pages/LikedFoodPage.js';
+import UnLikedFoodPage from './pages/UnLikedFoodPage.js';
 
 function App() {
   return (
-    <div className="App">
-      <LikedFoodPage />
-    </div>
+    <ErrorBoundary FallbackComponent={ErrorFallback}>
+      <div className="App">
```
Comment (Quaquaro) [inline src/App.js:10]: Do you really need the className="App"

File: src/pages/LikedFoodPage.js [hunk 3]
```diff
@@ -29,12 +30,15 @@ export default function LikedFoodPage() {
...
+      </footer>
     </LikedFoodPageStyle>
   );
 }
 
 const LikedFoodPageStyle = styled.section`
-  background-color: var(--peach);
+  background-color: var(--white);
   display: grid;
   grid-template-rows: 10vm 1fr 50px;
```
Comment (Quaquaro) [inline src/pages/LikedFoodPage.js:43]: What does vm stand for? vw viewport width?

File: src/pages/UnLikedFoodPage.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+export default function UnLikedFoodPage() {
+  return (
+    <LikedFoodPageStyle>
+      <Header>
+        <h1>Ich futtere lieber etwas anderes als...</h1>
+      </Header>
+      <main>
+        <ListStyle
+          data-testid={'unlikedlist'}
+          aria-describedby={
```
Comment (Quaquaro) [inline src/pages/UnLikedFoodPage.js:14]: I assume that you will create the list from an array with map in the near future.

**Review by Quaquaro — APPROVED** [review]
Keep up the good work.

File: src/components/Navigation.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  border: 2px solid var(--black);
+  min-height: 50px;
+  min-width: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  &.active {
+    background-color: ${'var(--steelblue)'};
+  }
+`;
```
Comment (PReichetanz) [inline src/components/Navigation.js:50]: It is the exact copy of ´NavButtonThumbup`, so you don't need it; just give it a more general name (e.g., NavButtonThumb) and you are good to go!

File: src/components/Navigation.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    </NavigationWrapper>
+  );
+}
+
+const NavigationWrapper = styled.section`
+  display: flex;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+`;
```
Comment (PReichetanz) [inline src/components/Navigation.js:24]: Wouldn't a ´\<nav\>` element be more semantical here?

**Review by PReichetanz — COMMENTED** [review]
Nice work, only had minor issues 🚀

File: src/App.js (hunk 2 above)
Comment (ttaannjjaa) [reply src/App.js:10]: Thanks for the advice - i will skip it

File: src/pages/LikedFoodPage.js (hunk 3 above)
Comment (ttaannjjaa) [reply src/pages/LikedFoodPage.js:43]: oh, typo, it should be `vmin`, thanks a lot

File: src/components/Navigation.js (hunk 5 above)
Comment (ttaannjjaa) [reply src/components/Navigation.js:50]: many thanks, i will change this

**Review by mwarnck — APPROVED** [review]
the little changes are looking good to me

**Pull request merged**